styles/about.css

```diff
@@ -0,0 +1,113 @@
+.hero {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 2.5rem;
+    max-width: 68.75rem;
+    margin: auto;
+  }
+  .hero img {
+    height: 25em;
+    width: 25rem;
+  }
+  
+  .bio {
+    width: 25rem;
+    padding: 0.625rem;
+    border-radius: 6px;
+    box-shadow: 0px 4px 15px 4px var(--primary-shadow);
+  }
+  
+  .bio h2 {
+    margin-bottom: var(--bottom-margin);
+  }
+  
+  .bio p {
+    line-height: var(--line-height);
+    padding: 0.3rem 0;
+  }
+
+  nav {
+    position: sticky;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    padding: .5rem 1rem;
+    background-color: var(--bg-color);
+    box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
+  }
+
+  .navbar a {
+    color: black;
+    padding: 14px 20px;
+    text-decoration: none;
+    display: inline-block;
+
+  }
+
+.navbar a:hover {
+    color: #0000;
+    background: 
+      linear-gradient(45deg, white, 50%,#000 0) 
+      var(--_p,100%)/200% no-repeat;
+    -webkit-background-clip: text;
+            background-clip: text;
+    transition: .5s;
+    --_p: 0%;
+  }
+
+
+
+  .fa {
+    padding: 15px;
+    font-size: 10px;
+    width: 7.5px;
+    height: 7.5px; 
+    text-align: center;
+    text-decoration: none;
+    border-radius: 50%;
+  }
+  
+
+  .fa:hover {
+    opacity: 0.7;
+  }
+  
+
+  
+  .fa-linkedin {
+    background: black;
+    color: white;
+  }
+  
+
+  .fa-instagram {
+    background: black;
+    color: white;
+  }
+
+  .fa-github {
+    background: black;
+    color:white
+  }
```

Comment:
Since these all have the same property, you can do
```
.fa:hover, .fa-instagram, .fa-github {

}
```
or give all of these elements the same class name instead of three separate ones.